### PR TITLE
Make entire workspace avatar clickable

### PR DESCRIPTION
**Pull request by Julesssss**
### Details

image

Remove an inconsistency in our workspace edit icon and avatar behavior. The whole avatar on the left is clickable but only the pencil edit icon on the right is. In addition, the pencil icon on the right in the Edit Workspace pane is misleading as clicking it actually opens the change profile pic option.

https://user-images.githubusercontent.com/10736861/135444110-1e117c6c-98c3-4338-9f64-cf7c79d16117.mov

### Fixed Issues

$ https://github.com/Expensify/App/issues/5537

### Tests

- As a new or existing user, create or navigate to a workspace
- Tap the Workspace Avatar to open the Edit Workspace page
- On this page, the Avatar should have a Camera icon next to it
- You should be able to click the Avatar OR Camera icon to upload an image
- When the image is uploading, the Avatar/Icon should not be pressable
- Once complete, tapping the Avatar/Icon should show an additional option to remove the image
- If clicked, the Avatar should reset to the default image

### QA Steps

Run above test

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

Screenshot 2021-09-30 at 12 03 52

#### Mobile Web

Screenshot_1633000300

#### Desktop

Screenshot 2021-09-30 at 12 12 18

#### iOS

Simulator Screen Shot - iPod touch (7th generation) - 2021-09-30 at 12 50 11

#### Android

Screenshot_1633001385

### CODE REVIEW

**Comment by Julesssss** [comment]
WIP, as [there is a chance that this design will change](https://github.com/Expensify/App/issues/5537#issuecomment-929381464) once we switch to the new workspace editor page.

File: src/pages/workspace/WorkspaceSidebar.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ const WorkspaceSidebar = ({
                                 <Tooltip absolute text={translate('workspace.common.edit')}>
                                     <View style={[styles.smallEditIcon, styles.smallAvatarEditIcon]}>
                                         <Icon
-                                            src={Pencil}
+                                            src={Camera}
```
Comment (MariaHCD) [inline src/pages/workspace/WorkspaceSidebar.js:123]: Shouldn't this be the `Pencil` icon and the icon in `AvatarWithImagePicker` be the `Camera` icon? This is what I'm seeing:
Screen Shot 2021-10-01 at 12 00 00 PM

**Review by MariaHCD — COMMENTED** [review]
Looks good and tests well! Just a question about the icons.

File: src/pages/workspace/WorkspaceSidebar.js (hunk 1 above)
Comment (shawnborton) [reply src/pages/workspace/WorkspaceSidebar.js:123]: I agree with you, those look like they got mixed up.

Comment (Julesssss) [reply src/pages/workspace/WorkspaceSidebar.js:123]: Wow, my bad. I switched the icons a couple of times and didn't notice this 😬 

Thanks for spotting.

Comment (Julesssss) [reply src/pages/workspace/WorkspaceSidebar.js:123]: Oh I see... I made the change locally, took the screenshots and then forgot to make the final commit.

**Comment by Julesssss** [comment]
Resolved, please take another look 🙇

**Review by MariaHCD — APPROVED** [review]
LGTM!

Screen Shot 2021-10-01 at 4 44 09 PM

**Review by luacmartins — APPROVED** [review]
LGTM and tests well!

**Pull request merged**